jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -114,10 +114,10 @@ function configure_controller_access {
         # url
         local kieServerControllerUrl=$(build_simple_url "${kieSererControllerProtocol}" "${kieServerControllerHost}" "${kieServerControllerPort}" "${kieServerControllerPath}")
         JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller=${kieServerControllerUrl}"
+        # user/pwd
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.user=\"$(get_kie_server_controller_user)\""
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.pwd=\"$(esc_kie_server_controller_pwd)\""
     fi
-    # user/pwd
-    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.user=\"$(get_kie_server_controller_user)\""
-    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.pwd=\"$(esc_kie_server_controller_pwd)\""
     # token
     local kieServerControllerToken="$(get_kie_server_controller_token)"
```

Comment:
Shouldn't the token also get moved up alone with the user and pwd? It is just another way to authenticate - alternative to user/pwd) if a controller (host) is specified.

Reply:
Same as above

Reply:
Same as above

Reply:
@errantepiphany Updated as per suggestion